server/auth/types/multiple/multi_auth.ts

```diff
@@ -174,9 +174,7 @@ export class MultipleAuthentication extends AuthenticationType {
         },
       });
     } else {
-      return response.unauthorized({
-        body: `Authentication required`,
-      });
+      return response.unauthorized();
```

Comment:
why this can solve the login timeout issue?

Reply:
Addressed.

Reply:
Addressed.